CHANGELOG.asciidoc

```diff
@@ -3,6 +3,44 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.6.1]]
+=== Beats version 8.6.1
+https://github.com/elastic/beats/compare/v8.6.0\...v8.6.1[View commits]
+
+==== Breaking changes
+
+*Winlogbeat*
+
+- Corrects issue with security events with source IP of "LOCAL" or "Unknown" failing to ingest
```

Comment:
This doesn't seem like a real breaking change. It also doesn't link to an issue or PR explaining what it is for.